cmd/utils/flags.go

```diff
@@ -141,6 +142,11 @@ var (
 		Usage: "Explicitly set network id (integer)(For testnets: use --ropsten, --rinkeby, --goerli, --kotti, --mordor, --yolov1 instead)",
 		Value: eth.DefaultConfig.NetworkId,
 	}
+	EthProtocolsFlag = cli.StringFlag{
+		Name:  "ethprotocols",
```

Comment:
What about `eth.protocols`? It seems like a general pattern is `<service>.<value>`.. ?